src/components/submission/data-row.vue

```diff
@@ -32,7 +33,7 @@ except according to the terms contained in the LICENSE file.
         <span class="encryption-overlay"></span>
       </td>
     </template>
-    <td :title="submission.__id">{{ submission.__id }}</td>
+    <td>{{ submission.__id }}</td>
```

Comment:
The instance ID column can go wider than others (it has a larger `max-width`), so it shouldn't need a tooltip.